Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -899,6 +899,27 @@
   - type: BorgModuleIcon
     icon: { sprite: Interface/Actions/actions_borg.rsi, state: clowning-module }
 
+- type: entity
+  id: BorgModuleAdvancedClowning
+  parent: [ BaseBorgModuleService, BaseProviderBorgModule ]
+  name: advanced clowning cyborg module
+  description: Advanced service module for only the silliest cyborgs! Comes with a built-in oven that bakes pies automatically over time, a holopeel projector, and a push horn.
+  components:
+  - type: Sprite
+    layers:
+    - state: service
+    - state: icon-clown-adv
+  - type: ItemBorgModule
+    items:
+    - BananiumHorn
```

Comment:
I know that this item is craftable for regular crew but as someone who has abused one to the point of being exploded by an admin, it is probably too much as part of the module. The push horn already works as a "horn upgrade" here.